earth_enterprise/rpms/opengee-fusion/post-uninstall.sh

```diff
@@ -26,4 +26,7 @@ remove_users_and_groups()
 # Main Function
 #-----------------------------------------------------------------
 
-remove_users_and_groups
+# remove users only if actually uninstalling
+if [ "$1" = "0" ] ; then
+    remove_users_and_groups
```

Comment:
Suggestion: We may want to just print a message that the system administrator can delete `gefusionuser`, if no files or directories are using it.

Reply:
We could.  For now I killed the post-uninstalls entirely in gradle...